### ISSUE TITLE
Need an example of configuration of hives

### ISSUE THREAD

**Issue by Pyromanc3r** [ForestryMC/ForestryMC#2148]
Hi. I need an example of a configuration of how to disable beehives in a specific biome. I do not quite understand those settings. I need to remove the hives from biomes (mod Abyssalcraft).
I tried this:
                S:forest <
                    Darklands Forest
                    Dreadlands Forest
                 >
                S:meadows <
                    Darklands
                    Darklands Plains
                 >
But it does not work.
It was easy with the trees. It was necessary only to indicate the world. But with these hives I'm stuck...
Thanks.

**Comment by temp1011** [comment]
~~I think you need to use the registry name of the biome. You want something like `abyssalcraft:darklands_forest` and `abyssalcraft:dreadlands_forest`. You can see the registry names assigned to the biomes here: https://github.com/Shinoow/AbyssalCraft/blob/8eabf832ac0cfc4f8fa4901505029e573abc63f1/src/main/java/com/shinoow/abyssalcraft/init/WorldHandler.java#L55-L69. The backpack folder should give a good idea of how things should look.~~

**Comment by temp1011** [comment]
This isn't possible yet.

**Comment by temp1011** [comment]
Actually this is possible. I'm just being very confused. It's a bit annoying though because you have to add it to every hive entry that generate's there. Still use the registry name. I'll look into making something to just blacklist all hives.

**Comment by temp1011** [comment]
Here is something that should blacklist all the biomes from abyssalcraft from all hives https://gist.github.com/temp1011/7e954d368bd2cbf7514f7be67bef168f. Let me know if you have any issues.

**Comment by Pyromanc3r** [comment]
Thanks for trying to help, but this, too, unfortunately does not work.
In principle, this is not a big problem. I'm considering the option to manually clean the world in creative mode (using Chunk Pregenerator) before installing map on my server.
And I see this option as the only correct one. Because all the same the same trees (which are can by disabled only in dimensions, but not in biomes) will remain in Overworld in Darklands and etc. Plus I want to install Botania which also does not have spawn settings.
(just in case: forestry-1.12.2-5.8.0.291, AbyssalCraft-1.12.2-1.9.4.9)

**Comment by Pyromanc3r** [comment]
It seems I found a solution. Using the mod "Recurrent Complex" I found a forge biome type.
SPOOKY
DEAD
I put it in blacklist in all category (although I'm not sure that we need all, need more testing). Hives are no longer in biomes Abyssalcraft
It's done.
But there are still trees... Which can be disabled only in the Dimensions, but not in biomes.
Need new section in Forestry config (similar to hives)
